Attendance_app.py

```diff
@@ -285,6 +301,31 @@ def exit(self):
         self.controller.destroy()
         self.controller.quit()
 
+    # Autocompletion Helper Functions
+    def match_string(self):
+        hits = []
+        classVar = self.class_variable.get()

```

Comment:
⚠️ `camelCase usage for variable name`
Please use snake_case for variable names.